api/src/Common/Models/RaceResult.cs

```diff
@@ -19,7 +19,6 @@ public struct RaceResult : IModel
         public string Comments { get; set; }
 
         public string DataSource { get; set; }
-
```

Comment:
Was this intentional?

Reply:
no - accidentally deleted when I was testing an attribute here